wordpress-mobile/WordPress-Android
# Prompt for Jetpack setup from Notifications

* Connect Jetpack flow to Notification tab when user is not logged in with their WordPress.com account. 
* Some renaming and refactoring 
* Issue: #7198 

To test:
* Log in with self-hosted page without WordPress.com account
* Go to Notifications tab
* See Setup Jetpack prompt
* Finish Jetpack flow
* See Notifications

![screenshot_1518445478](https://user-images.githubusercontent.com/1079756/36101204-f22d8472-1008-11e8-8051-94b6f881ab3d.png)
